OpenSim/Sandbox/ExampleHopperDevice/exampleHopperDevice.cpp

```diff
@@ -254,11 +259,12 @@ int main()
         addSignalGeneratorToDevice(*device);
 
         // List the device outputs we wish to display during the simulation.
-        std::vector<std::string> deviceOutputs{ "length", "tension", "power",
-                                                "controller/myo_control" };
+        std::vector<std::string> deviceOutputs{ "length", "tension", "power" };
+        std::vector<std::string> controllerOutputs{ "myo_control" };
```

Comment:
I think this should be `deviceControllerOutputs` to reinforce the ownership tree (i.e., that the controller is owned by the device).